packages/template-parser/src/index.ts

```diff
@@ -1,5 +1,5 @@
 import { parseTemplate, ParseSourceSpan } from '@angular/compiler';
-import EventEmitter from 'events';
```

Comment:
Hmm why did this change?

Reply:
Umm this was an intentional change because `yarn build` didn’t succeed. I will paste an error here when I get back to PC.

Reply:
@JamesHenry 

![image](https://user-images.githubusercontent.com/7337691/101812172-b0245f80-3b23-11eb-8a02-96e312cc5198.png)

This happens when the `EventEmitter` is imported via the default import.

Reply:
I have reverted the change to `import EventEmitter`, this seems like `yarn` accidently updated `@types/node` in my workspace to the latest version, because the latest `@types/node` version doesn't allow to import `EventEmitter` via default import.